src/client/character_creation.ts

```diff
@@ -0,0 +1,73 @@
+import * as display from './display.js';
+import { Vector2 } from '../common/vector2.js';
+import { HudPiece, Anchor, Tile, Drawable } from './drawing/drawable.js';
+import { StaticImage, LoopingImage, Image } from './drawing/image.js';
+
+let position: Vector2 | null = null;
+let curr = 2;
+let img = new Image();
+let presets: Array<string>;
+presets = [
+  '../assets/gandalf_the_girthy_green.png',
+  '../assets/gandalf_the_girthy_blue.png',
+  '../assets/gandalf_the_girthy_red.png',
+  '../assets/gandalf_the_girthy_purple.png',
+];
+window.requestAnimationFrame(creationInit);
+function creationInit(totalMilliseconds: number): void {
+  display.draw((context) => {
+    context.imageSmoothingEnabled = false;
+    drawPreset(context);
+    let arrowLeft = new Image();
+    arrowLeft.src = '../assets/arrow_left.png';
+    arrowLeft.onload = function () {
+      context.drawImage(arrowLeft, 0, 256, 128, 64);
+    };
+    let arrowRight = new Image();
+    arrowRight.src = '../assets/arrow_right.png';
+    arrowRight.onload = function () {
+      context.drawImage(arrowRight, 128, 256, 128, 64);
+    };
+  });
+  display.onMouseMove((event) => {
+    position = event.position;
+    
+    console.log(`position: ${position?.x} ${position?.y}`);
+  });
+  display.onMouseUp((event) => {
+    console.log(`Mouse up position: ${position?.x} ${position?.y}`);
+    if (isLeftArrowClicked()) {
+      curr = (curr - 1) % 4;
```

Comment:
> curr = (curr + 3) % 4;

Reply:
will be fixed in the next relea... ekhm, commit